src/V3File.cpp

```diff
@@ -307,7 +307,7 @@ void V3File::writeTimes(const string& filename, const string& cmdlineIn) {
 bool V3File::checkTimes(const string& filename, const string& cmdlineIn) {
     return dependImp.checkTimes(filename, cmdlineIn);
 }
-void V3File::createMakeDirFor(const string& filename) {
+void V3File::createMakeDirFor(const string& filename) VL_MT_SAFE {
```

Comment:
This calls createMakeDir, which isn't marked safe.  Why did t_a5_attributes_src.pl pass?

Reply:
``createMakeDir`` was marked as safe in header file. Unfortunately we can only check if there is missing annotation in ``declaration`` (for ``definition`` clang returns annotations from both ``declaration`` and ``definition``, for ``declaration`` clang returns annotations only from ``declaration``).

I think this is also result of bad rebase. I'm planning to make this MT safe in next PR that includes locking. For now I've fixed annotations and left this in unsafe calls.

Reply:
Ok.  Maybe it would make sense to make a minimal patch to get all the exceptions out of t_a5_attributes_src.pl, then once that is in consider adding the additional VL_MT_SAFE's etc.